convert/sentencepiece_model.proto

```diff
@@ -0,0 +1,333 @@
+// Copyright 2016 Google Inc.
```

Comment:
I think no is totally a good answer, but should we add a `go generate` for this or a comment on how to compile the protobuf?

Adding go generate means running `go generate ./...` will cause everyone working on the project to need protoc, which isn't ideal

Reply:
I added a comment on how to do this. We shouldn't have to regenerate this for the most part, so requiring everyone to run `protoc` all the time seems not ideal.